### PR TITLE
[SofaMiscMapping] Clean DeformableOnRigidFrameMapping

### PR DESCRIPTION
**Pull request by olivier-goury**
Simple PR to clean up DeformableOnRigidFrameMapping, during DEFROST team coding friday.

______________________________________________________

This PR: 
- [ ] builds with SUCCESS for all platforms on the CI.
- [ ] does not generate new warnings.
- [ ] does not generate new unit test failures.
- [ ] does not generate new scene test failures.
- [ ] does not break API compatibility.
- [ ] is more than 1 week old (or has fast-merge label).

**Reviewers will merge only if all these checks are true.**

### CODE REVIEW

**Comment by epernod** [comment]
[ci-build][with-all-tests]

File: modules/SofaMiscMapping/DeformableOnRigidFrameMapping.inl [hunk 1]
```diff
@@ -104,7 +87,7 @@ void DeformableOnRigidFrameMapping<TIn, TInRoot, TOut>::init()
     if(!this->getFromModels2().empty())
     {
         m_fromRootModel = this->getFromModels2()[0];
-        sout << "Root Model found : Name = " << m_fromRootModel->getName() << sendl;
+        sout << "Root Model found : Name = " << m_fromRootModel->getName() ;
```
Comment (epernod) [inline modules/SofaMiscMapping/DeformableOnRigidFrameMapping.inl]: you missed one!

File: modules/SofaMiscMapping/DeformableOnRigidFrameMapping.inl [hunk 2]
```diff
@@ -473,234 +506,11 @@ void DeformableOnRigidFrameMapping<TIn, TInRoot, TOut>::handleTopologyChange(cor
...
-//		//	// in case of a "multi-mapping" (the articulation system is placed on a  simulated object)
-//		//	// the constraints are transmitted to the rootModle (the <rigidtype> object which is the root of the articulated system)
-//		//	if (m_fromRootModel!=NULL)
-//		//		m_fromRootModel->setConstraintId(*it);
-//		//	it++;
-//		//}
-//	}
-//}
-
-/*
```
Comment (epernod) [inline modules/SofaMiscMapping/DeformableOnRigidFrameMapping.inl:672]: what a huge commented code!

File: modules/SofaMiscMapping/DeformableOnRigidFrameMapping.inl [hunk 3]
```diff
@@ -473,234 +506,11 @@ void DeformableOnRigidFrameMapping<TIn, TInRoot, TOut>::handleTopologyChange(cor
...
-
-
-
-
-
-sofa::core::objectmodel::
-m_fromModel->getContext()->get(m_fromRootModel, core::objectmodel::BaseContext::SearchUp);
-
-
+    // (30-11-2018, Olivier Goury): Not implemented yet. Someone want to do it?
```
Comment (epernod) [inline modules/SofaMiscMapping/DeformableOnRigidFrameMapping.inl:512]: you put your name...

File: modules/SofaMiscMapping/DeformableOnRigidFrameMapping.inl (hunk 1 above)
Comment (olivier-goury) [reply modules/SofaMiscMapping/DeformableOnRigidFrameMapping.inl]: Indeed, thanks!

**Review by epernod — APPROVED** [review]

**Pull request merged**